.pipelines/templates/android_shared_lib_build.yml

```diff
@@ -0,0 +1,70 @@
+# packaging pipeline for onnxruntime-extensions Android AAR in nuget package
+parameters:
+- name: AndroidABI
+  displayName: android abi,[arm64-v8a, x86_64, X86, armeabi-v7a]
+  values:
+  - 'x86_64'
+  - 'x86'
+  - 'arm64-v8a'
+  - 'armeabi-v7a'
+  type: string
+- name: JobSuffix
+  displayName: android abi has dash `-`, which can't be part of job name.
+  type: string
+- name: AdditionalBuildFlags
+  displayName: set OCOS_ENABLE_CV2=OFF OCOS_ENABLE_BLINGFIRE
+  type: string
+
+
+jobs:
+  - job: Android_C_API_Packaging_${{ parameters.JobSuffix }}
+    pool:
+      vmImage: "macOS-12"
+    timeoutInMinutes: 120
+    variables:
+      buildConfig: Release
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.9"
+          addToPath: true
+          architecture: "x64"
+        displayName: "Use Python 3.9"
+
+      - task: JavaToolInstaller@0
+        displayName: Use jdk 11
+        inputs:
+          versionSpec: "11"
+          jdkArchitectureOption: "x64"
+          jdkSourceOption: "PreInstalled"
+
+      - script: brew install coreutils ninja
+        displayName: Install coreutils and ninja
+
+      - task: Bash@3
+        displayName: 'Build android shared library'
+        inputs:
+          targetType: 'inline'
+          script: |
+            set -e -x
+            python ./tools/android/build_aar.py \
+                --output-dir $(Build.BinariesDirectory)/android_so \
+                --build_target so       \
+                --abi ${{ parameters.AndroidABI }}       \
+                --config $(buildConfig) \
+                --cmake-extra-defines ${{parameters.AdditionalBuildFlags}}
```

Comment:
from the name, it's not obvious that "AdditionalBuildFlags" are starting the value of `--cmake-extra-defines`. one could pass additional, unrelated options too after that.

maybe call it "CmakeExtraDefines" and pass it like `--cmake-extra-defines "${{ parameters.CmakeExtraDefines }}"`?

similar comment for other pipeline files